samples/durable-task-sdks/dotnet/Agents/PromptChaining/Worker/Orchestrations/ContentCreationOrchestration.cs

```diff
@@ -0,0 +1,67 @@
+using AgentChainingSample.Activities;
+using AgentChainingSample.Shared.Models;
+using Microsoft.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace AgentChainingSample.Orchestrations;
+
+/// <summary>
+/// Orchestrates the news article generation workflow with multiple specialized agents
+/// </summary>
+public class ContentCreationOrchestration
+{
+    private readonly ILogger<ContentCreationOrchestration> _logger;
+
+    public ContentCreationOrchestration(ILogger<ContentCreationOrchestration> logger)
```

Comment:
Is this a "replay safe" logger?  AFAIK, that's what you get when you use the logger from the `TaskOrchestrationContext`, so you may not need to inject it.